CMakeLists.txt

```diff
@@ -293,7 +293,9 @@ if ( ENABLE_CPACK )
     elseif ( ${CMAKE_SYSTEM_NAME} STREQUAL "Linux" )
       include ( CPackLinuxSetup )
       # let people know what is coming out the other end
-      message ( STATUS "CPACK_PACKAGE_FILE_NAME = ${CPACK_PACKAGE_FILE_NAME}" )
+      if (CPACK_GENERATOR STREQUAL "DEB") # rpm filename is not visible?
```

Comment:
If you search for `CPACK_PACKAGE_FILE_NAME` in http://builds.mantidproject.org/view/Release%20Pipeline/job/release_clean-rhel7/183/consoleText then it comes up. Can we keep this? I think it's quite useful to see the filename at configure time.